Automattic/wp-calypso
# Pricing page rework: Extract items list component logic for re-usability

#### Proposed Changes

* This PR extracts some code related to items list rendering for re-usability as suggested in https://github.com/Automattic/wp-calypso/pull/67433#discussion_r964373946

#### Testing Instructions

- Do any one of these
    * Click on Jetpack Cloud live link below and wait for the redirect to complete, then goto `/pricing?flags=jetpack/pricing-page-rework-v1`
    * or boot up this PR 
        * Run `git fetch && git checkout update/jp-pricing-rework/extract-items-list-code`
        * Run `yarn start-jetpack-cloud`
        * Goto [http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1](http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1)
- Confirm that everything is same as before

#### Pre-merge Checklist







<img width="1342" alt="Screenshot 2022-09-07 at 5 16 32 PM" src="https://user-images.githubusercontent.com/18226415/188870947-f043fc3b-4316-4899-a648-d2603c62009a.png">
